### ISSUE TITLE
Integrations can not be activated through backend

### ISSUE THREAD

**Issue by ghost** [magento/magento2#32542]
### Preconditions (*)

1. Magento 2.4.2

### Steps to reproduce (*)

1. Have an integration that is deactivated
2. Go to system->integrations (admin/integration/index/)
3. Press "activate" on an existing deactivated integration

### Expected result (*)

I would expect that this would activate the integration.

### Actual result (*)

This does not activate the integration. It creates a GET request to the TokensExchange controller. However the controller only accepts POST requests. This leads to an unexpected error because the request actually responds with a 404 page.
![billede](https://user-images.githubusercontent.com/78018685/111597606-9fdcea00-87ce-11eb-8b54-ad9e52aef7ea.png)

As a temporary solution I "hacked" in the following line in the controller and everything worked as expected afterwards:
```
class TokensExchange extends Integration implements HttpPostActionInterface,HttpGetActionInterface
```

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.4/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [X] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Oscar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-41427` were created

**Issue Available**: @engcom-Oscar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by adamzero1** [comment]
@magento I am working on this

**Comment by adamzero1** [comment]
This is already fixed by: https://github.com/magento/magento2/commit/68d20230ca9d6198fa3093eb174e24aff47b1646

**Comment by gabrieldagama** [comment]
@engcom-Oscar, can you re-confirm this issue? It seems to be fixed.

Thanks!

**Comment by engcom-Oscar** [comment]
@gabrieldagama  I still can reproduce it on latest 2.4-develop. Button "Activate" does not work to activate inactive integration.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Oscar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-41427` were created

**Issue Available**: @engcom-Oscar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @sport24-rtr. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-41427` was closed as `non-reproducible` in `2.4-develop`.
It means that Magento team either unable to reproduce this issue using provided _Steps to Reproduce_ from the **_Description section_** on clean Magento instance or the issue has been already fixed in the scope of other tasks.

But if you still run into this problem please update or provide additional information/steps/preconditions in the **_Description section_** and reopen this issue.

**Comment by netjordan** [comment]
Hello,

I am facing the same issue as the original person described, and confirmed that their temporary fix does solve the issue. This is on the 2.4.2 release version.

Happy to provide further info etc if it helps with re-production?

Thanks,
Jordan

**Comment by gjportegies** [comment]
Issue still persist in Magento 2.4.2-p1

**Comment by sport24-rtr** [comment]
@gjportegies @netjordan 
FYI: https://github.com/magento/magento2/commit/68d20230ca9d6198fa3093eb174e24aff47b1646 has been released with 2.4.3 which should fix the issue.
